test/test_quantize_script.py

```diff
@@ -71,5 +72,101 @@ def forward(self, x):
                    .check_not('Observer = prim::GetAttr[name="_observer_') \
                    .run(m.graph)
 
+
+    def test_insert_quant_dequant_dynamic(self):
```

Comment:
rename this to `test_insert_quant_dequant_dynamic_conv`? since you have dynamic linear as well